packages/fluent-editor/src/table/better-table.ts

```diff
@@ -284,16 +318,16 @@ class BetterTable extends Module {
   }
 
   handleCompositionend(quill) {
-    // fix: 解决在空行行首用中文输入法输入字符，然后按退格导致插入多余空行的问题
+    // fix: 解决在空行行首用中文输入法输入字符，然后按格导致插入多余空行的问题
     const range = quill.getSelection()
-    const [line] = quill.getLine(range.index)
+    const [ line ] = quill.getLine(range.index)
     const domNode = line.domNode
     const isInParagraph = domNode.nodeName === 'P'
     const isInTableCell = line.statics.blotName === 'table-cell-line'
     if ((isInParagraph || isInTableCell)
       && domNode.childNodes.length > 1
-      && domNode.childNodes[0].nodeName === 'BR') {
-      domNode.childNodes[0].remove()
+      && domNode.childNodes[ 0 ].nodeName === 'BR') {
+      domNode.childNodes[ 0 ].remove()
```

Comment:
@jany55555 中括号两边加上空格这个格式不太对，我看多处有这个问题。
```suggestion
      domNode.childNodes[0].remove()
```